external-crates/move/move-compiler/tests/move_check/parser/function_visibility_empty.exp

```diff
@@ -2,5 +2,5 @@ error[E01002]: unexpected token
   ┌─ tests/move_check/parser/function_visibility_empty.move:2:5
   │
 2 │     public() fun f() {}
-  │     ^^^^^^^^ Invalid visibility modifier. Consider removing it or using 'public' or 'public(friend)'
+  │     ^^^^^^^^ Invalid visibility modifier. Consider removing it or using 'public(friend)', 'public(package)' or 'public(script)'
```

Comment:
Probably also want to add-back/keep `public` as a suggestion in this error message as well.

Reply:
Changed this and regenetated.